Permanent/Views/Cells/FileDetailsCollectionViewCells/FileDetailsMapViewCellCollectionViewCell.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  FileDetailsMapViewCellCollectionViewCell.swift
+//  Permanent
+//
+//  Created by Lucian Cerbu on 15.03.2021.
+//
+
+import UIKit
+import MapKit
+
+class FileDetailsMapViewCellCollectionViewCell: UICollectionViewCell {
+
+    static let identifier = "FileDetailsMapViewCellCollectionViewCell"
+    let regionInMeters: Double = 5000
+    
+    @IBOutlet weak var titleLabelField: UILabel!
+    @IBOutlet weak var detailsTextField: UITextField!
+    @IBOutlet weak var locationMapView: MKMapView!
+    
+    static func nib() -> UINib {
+        return UINib(nibName: identifier, bundle: nil)
+    }
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        
+        detailsTextField.delegate = self
+    }
+
+    func configure(title: String, details: String, isDetailsFieldEditable: Bool = false) {
+        titleLabelField.text = title
+        titleLabelField.textColor = .white
+        titleLabelField.font = Text.style9.font
+
+        detailsTextField.text = details
+        detailsTextField.backgroundColor = .clear
+        detailsTextField.textColor = .white
+        detailsTextField.font = Text.style8.font
+        detailsTextField.isUserInteractionEnabled = isDetailsFieldEditable
```

Comment:
will this be editable?

Reply:
Will add another screen when user taps on the location cell, in this screen a map is shown with a text field, where the user can input the new location.